### ISSUE TITLE
Set character limit on question description

### ISSUE THREAD

**Issue by CaduRoriz** [TEAMMATES/teammates#12460]
Inside feedback editing functionality, in the editing question, it doesn't have character limit to the description question, as well as in the configuration, it can cause problems to the user. I haven't checked but I believe that this problem is not exclusive to the question and configuration editing session, I believe that in creation there is no limit either.

![image](https://github.com/TEAMMATES/teammates/assets/77363222/fe736d5d-093c-4999-8312-a884c713eccf)

**Comment by samuelfangjw** [comment]
> it can cause problems to the user

@CaduRoriz what problems exactly?

**Comment by wkurniawan07** [comment]
@samuelfangjw although it might not be what the issue reporter is describing, we do need to have some limit on the backend (see https://github.com/TEAMMATES/teammates/issues/12455#issuecomment-1564127993)

**Comment by samuelfangjw** [comment]
> @samuelfangjw although it might not be what the issue reporter is describing, we do need to have some limit on the backend (see [#12455 (comment)](https://github.com/TEAMMATES/teammates/issues/12455#issuecomment-1564127993))

Yes, that's a valid concern. Another concern here is that the issue suggests that there are user-facing problems related to this issue. Would like to know what exactly from @CaduRoriz  so we can discuss and triage this potential issue properly.

**Comment by Kingsterdam** [comment]
Is this issue Available, I want to work on it ?

**Comment by weiquu** [comment]
Hi @Kingsterdam, feel free to submit a PR for this issue

**Comment by khushboo-9876** [comment]
Is this issue Available, I want to work on it ?

**Comment by domlimm** [comment]
@Kingsterdam are you working on this? If not @khushboo-9876 will take this. Thanks!

**Comment by sawilkhan** [comment]
Is someone working on it or can i?

**Comment by domlimm** [comment]
Since @Kingsterdam has not replied, @khushboo-9876 are you still interested in taking this issue?

Do let us know, if not, we will have @sawilkhan take it.

Thanks!

**Comment by anguye68** [comment]
Hello, I can work on this issue if it's available since there has not been any updates in a month.

**Comment by weiquu** [comment]
Hi @anguye68, feel free to submit a PR for this

**Comment by u7718594** [comment]
Hi, I can work on this issue if its available.

**Comment by 090997** [comment]
Hi, can I try to work on this issue?

**Comment by erinmayfield** [comment]
Hi, could I please work on this issue? 
(u7317656, w10)

**Comment by Miranda-lrx** [comment]
Hi, I'd like to work on it.
U7763307 workshop10

**Comment by cedricongjh** [comment]
> Hi, I can work on this issue if its available.

hi @u7718594, feel free to submit a PR for this issue

@SUPWEZZ, @090997, @erinmayfield, @Miranda-lrx since @u7718594 commented first, we'll let them submit a PR for it first unless they do not want to work on it anymore

**Comment by anjum-edge** [comment]
Set the character limit to 1000 words to get it done without any problems because the more words you add the more irrelevent it becomes.
Maintain the simplicity that's it
